cleansio/censor/censor_realtime.py

```diff
@@ -1,14 +1,104 @@
 """ Censors audio chunks in a continuous stream """
```

Comment:
There needs to be a mac class for realtime censoring. I have most of it done, I'll add it with you in person